test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -340,8 +340,8 @@ public void LightGbmMulticlassEstimatorCorrectSigmoid()
             var model = trainer.Fit(transformedDataView, transformedDataView);
 
             // The slope in the all the calibrators should be equal to the negative of the sigmoid passed into the trainer.
-            Assert.True(model.Model.SubModelParameters.All(predictor =>
-            ((FeatureWeightsCalibratedModelParameters<LightGbmBinaryModelParameters, PlattCalibrator>)predictor).Calibrator.Slope == -sigmoid));
+            // Since the Model/SubModelParameters are strongly typed, no cast is needed.
```

Comment:
This comment doesn't appear to add value after this change. Someone in a few months will read this, and not understand why it is talking about casting.

Reply:
Good point. I hadn't thought about it like that before. Removed that comment.